common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -45,6 +45,13 @@ public class RssClientConf {
       .withDescription("The type of partition shuffle data distribution, including normal and local_order. "
           + "The default value is normal. This config is only valid in Spark3.x");
 
+  public static final ConfigOption<Integer> MAX_CONCURRENCY_PER_PARTITION_TO_WRITE = ConfigOptions
+      .key("rss.client.max.concurrency.per-partition.write")
+      .intType()
+      .defaultValue(0)
```

Comment:
Do we need checkValue? If we use multi threads to write multiple disks local file, do we need respect this config option?

Reply:
> Do we need checkValue?

By default, the value is <= 0, it will be ignored.

> If we use multi threads to write multiple disks local file, do we need respect this config option?

Yes. So if specifying different values for different storage tiers, the extra option is needed.